python/whylogs/core/metrics/metrics.py

```diff
@@ -291,7 +291,7 @@ def columnar_update(self, view: PreprocessedColumn) -> OperationResult:
                 n_b = len(lst)
                 if n_b > 1:
                     mean_b = statistics.mean(lst)
-                    m2_b = statistics.pvariance(lst) * (n_b - 1)
+                    m2_b = statistics.pvariance(lst) * (n_b)
```

Comment:
why call `pvariance()` instead of `variance`?

Reply:
I thought changing the function called is a larger risk change, and either works as long as we have the appropriate normalization

Reply:
Seems better to call the function that actually computes the one we want :)

Reply:
It is only used in the update of m2 here, so what we want is something consistent with the update, but sure, lets keep it consistent with using sample variance. Updated.